### ISSUE TITLE
Rowspan with nested table or columns not spanning the rows

### ISSUE THREAD

**Issue by SaWey** [bpampuch/pdfmake#997]
When trying to nest a table inside a cell with a rowspan defined, the nested table is not rendered correctly.
It is not spanning the defined number of rows, it is only enlarging the current row.
Using columns generates the same result.

Here is my docdef:
```
var dd ={
    content : [{
            table : {
                widths : [80, 250, 170],
                body : [
                    [
                        {text : 'Row 1'}, {text : "Some text here"}, 
                        {rowSpan : 3,
                            columns : [
                                {text: "test data in rowspan"}, 
                                {text : " Bigger font", width : '*', margin : [10, 14],fontSize : 20,}
                            ]
                        }
                    ],
                    [{text : 'Row 2 title'}, {text : "Row 2 data"}, {}],
                    [{text : 'Row 3 title'}, {text : "Row 3 data"}, {}],
                ]
            }
        },
    ]
}
```

**Comment by devmarwen** [comment]
Any updates on this, we are experiencing the same issue.

**Comment by lerusal** [comment]
We are experiencing the same issue, are the any news about this issue?

**Comment by liborm85** [comment]
Connected with https://github.com/bpampuch/pdfmake/issues/1042

**Comment by pseisam** [comment]
Also experiencing this and wanted to add a screenshot of the behavior.
The rows Test1 and Test2 should have the same height, since the nested table has a rowSpan of 2. But instead the Test2 row is pushed down.
Has anyone found a workaround yet?

Code:

```
let content = [];
let nestedTable = {
    rowSpan: 2,
    table: { headerRows: 1, body: [["Nested1", "Nested2"], ["Nested3", "Nested4"]] }
}
content.push({
    table: { headerRows: 1, body: [["Test1", nestedTable], ["Test2", {}]] }
});
```

Screenshot:
![nestedTable](https://user-images.githubusercontent.com/11456503/55485697-3e480300-562b-11e9-865d-4afba7b3c16a.png)

**Comment by emidiotorre** [comment]
I am still experiencing this issue.
What is causing this behavior? 
How can we solve this? How can I help?

It is a real big problem for me right now, and i really need to make this work.
Thanks a lot!

**Comment by miroslavvojtus** [comment]
We face the very same issue.
Also it is quite a significant problem for us.

**Comment by FunctorDev** [comment]
Any updates on this?
It has been 4 years

**Comment by justlester** [comment]
is there any workaround on this?

**Comment by liborm85** [comment]
Fixed by PR https://github.com/bpampuch/pdfmake/pull/2764.
Released in version 0.2.11.
